src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -150,31 +160,56 @@ export class HuiDialogEditCard extends LitElement implements HassDialog {
         @keydown=${this._ignoreKeydown}
         @closed=${this._cancel}
         @opened=${this._opened}
-        .heading=${html`${heading}
-        ${this._documentationURL !== undefined
-          ? html`
-              <a
-                class="header_button"
-                href=${this._documentationURL}
-                title=${this.hass!.localize("ui.panel.lovelace.menu.help")}
-                target="_blank"
-                rel="noreferrer"
-                dir=${computeRTLDirection(this.hass)}
-              >
-                <mwc-icon-button>
-                  <ha-svg-icon path=${mdiHelpCircle}></ha-svg-icon>
-                </mwc-icon-button>
-              </a>
-            `
-          : ""}`}
+        .heading=${html`
```

Comment:
You should use the `heading` slot of `ha-dialog` <https://github.com/home-assistant/frontend/blob/dev/src/dialogs/more-info/ha-more-info-dialog.ts#L96>

Reply:
Or with a tab bar:
<https://github.com/home-assistant/frontend/blob/dev/src/panels/config/entities/dialog-entity-editor.ts#L95-L147>